src/main/java/org/jenkinsci/plugins/github/config/GitHubServerConfig.java

```diff
@@ -98,28 +91,13 @@ public GitHubServerConfig(String credentialsId) {
     }
 
     /**
-     * {@link #customApiUrl} field should be defined earlier. Because of we get full content of optional block,
-     * even if it already unchecked. So if we want to return api url to default value - custom value should affect
+     * Set the API endpoint.
      *
      * @param apiUrl custom url if GH. Default value will be used in case of custom is unchecked or value is blank
      */
     @DataBoundSetter
     public void setApiUrl(String apiUrl) {
-        if (customApiUrl) {
-            this.apiUrl = defaultIfBlank(apiUrl, GITHUB_URL);
-        } else {
-            this.apiUrl = GITHUB_URL;
-        }
-    }
-
-    /**
-     * Should be called before {@link #setApiUrl(String)}
-     *
-     * @param customApiUrl true if optional block "Custom GH Api Url" checked in UI
-     */
-    @DataBoundSetter
-    public void setCustomApiUrl(boolean customApiUrl) {
```

Comment:
See description of the field. This was used only for UI because of pure jelly form.